### DISCUSSION TITLE
Lesson 6 - Auto detect Ganache not happening.

### DISCUSSION THREAD

**Discussion by Dave777** [smartcontractkit/full-blockchain-solidity-course-py#936] [Q&A] [answered]
At 5:36:51 Patrick says to open up Ganache with Quick Start
and "quickly create our own Ganache blockchain"

I started Ganache and got the usual Accounts showing up,
but when I run brownie run scripts/deploy.py,  brownie does not see the local blockchain.

My ganache uses HTTP://127.0.0.1:7545

This is full response:

BrownieFundMeProject is the active project.

Launching 'ganache-cli.cmd --port 8545 --gasLimit 12000000 --accounts 10 --hardfork istanbul --mnemonic brownie'...

Running 'scripts\deploy.py::main'...
The active network is development
Deploying Mocks
Transaction sent: 0x4927014e712d84166b5af789150580173819b951a32ee5f30d8799353c875f09
  Gas price: 0.0 gwei   Gas limit: 12000000   Nonce: 0
  MockV3Aggregator.constructor confirmed   Block: 1   Gas used: 428967 (3.57%)
  MockV3Aggregator deployed at: 0x3194cBDC3dbcd3E11a07892e7bA5c3394048Cc87

Mocks Deployed
Transaction sent: 0x1b8a0bd530629cf0845d026459622172de8fdfcbd8a6d69206a9b37a255b4d7a
  Gas price: 0.0 gwei   Gas limit: 12000000   Nonce: 1
  FundMe.constructor confirmed   Block: 2   Gas used: 388393 (3.24%)
  FundMe deployed at: 0x602C71e4DAC47a042Ee7f46E0aee17F94A3bA0B6

Contract deployed to 0x602C71e4DAC47a042Ee7f46E0aee17F94A3bA0B6
Terminating local RPC client...

**Comment by Dave777** [comment, marked as answer]
I continued with the course,

It doesn't seem to matter as when I define ganache-local, it all works 👍

**Reply by cromewar** [reply]
Well you did a great job figuring that out, fell free to make any other questions.
